tests/unit/components/DocumentationTopic/PrimaryContent/Mentions.spec.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2024 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import { shallowMount } from '@vue/test-utils';
+import Mentions from 'docc-render/components/DocumentationTopic/PrimaryContent/Mentions.vue';
+import Mention from 'docc-render/components/DocumentationTopic/PrimaryContent/Mention.vue';
+
+function createReferences(names) {
+  return names.reduce((refs, name) => {
+    const id = 'doc://' + name;
+    refs[id] = {
```

Comment:
The linter is likely complaining about this (mutating a param). I don't mind if you just disable that rule for this specific instance since it's not harmful.

Otherwise, you can rewrite the reducer to appease the linter using something like the following pseudo code:
```
=> ({
  ...refs,
  [`doc://${name}`] = {...}
})
```